Gems/Atom/Tools/MaterialCanvas/Code/Source/Window/MaterialCanvasGraphView.h

```diff
@@ -12,10 +12,24 @@
 #include <AtomToolsFramework/Document/AtomToolsDocumentNotificationBus.h>
 #include <AtomToolsFramework/GraphView/GraphView.h>
 #include <AtomToolsFramework/Window/AtomToolsMainWindowRequestBus.h>
+#include <GraphCanvas/Components/Nodes/Comment/CommentBus.h>
+#include <GraphCanvas/Types/ConstructPresets.h>
 #endif
 
 namespace MaterialCanvas
 {
+    //! The implementation of the graph view requires construct presets in order to be able to create node groups and comment blocks.
+    class MaterialCanvasGraphConstructPresets : public GraphCanvas::EditorConstructPresets
+    {
+    public:
+        AZ_RTTI(MaterialCanvasGraphConstructPresets, "{8E349BC8-1D8B-4A1B-8DE0-FFD61438DBBD}", GraphCanvas::EditorConstructPresets);
+        AZ_CLASS_ALLOCATOR(MaterialCanvasGraphConstructPresets, AZ::SystemAllocator, 0);
+
+        MaterialCanvasGraphConstructPresets() = default;
+        ~MaterialCanvasGraphConstructPresets() = default;
```

Comment:
I believe that a non-virtual destructor in a derived class won't be called if `delete` is invoked through the class base pointer. This is a stateless class so it doesn't matter here, but usually things like this make me nervous since it might explode someday, perhaps consider adding the `override` postfix keyword?

Reply:
Adjusting follow up PR